src/ProjectService.Data/UserRepository.cs

```diff
@@ -106,5 +100,31 @@ public void Remove(Guid userId)
 
             _provider.Save();
         }
+
+        public bool AreExist(params Guid[] ids)
+        {
+            return AreExist(null, ids);
+        }
+
+        public bool AreExist(bool? isManager, params Guid[] ids)
```

Comment:
what is this?

Reply:
This is in order not to drag users to the command, but to check immediately in the repository

Reply:
rewrited